packages/data-stores/src/plans/test/resolvers.ts

```diff
@@ -116,7 +116,7 @@ describe( 'getSupportedPlans', () => {
 				{
 					annualPrice: '₹0',
 					billingPeriod: 'ANNUALLY',
-					pathSlug: undefined,
+					pathSlug: 'free',
```

Comment:
Ok to use a string directly here since the response is typed: https://github.com/Automattic/wp-calypso/blob/trunk/packages/data-stores/src/plans/types.ts#L34